profile/titles/en/README.md

```diff
@@ -60,3 +60,20 @@ It should also be noted that these criteria chiefly examine the scope of work so
 - Recognized widely in the industry for material contributions to the state of the art.
 - Invents new concepts, pushes the whole organization forward regularly.
 - Usually at least 15+ years of experience being a professional programmer in the specific domain.
+
+# Summary of salaries
+
+| level     | annual   | 2 year bonus | 5 year bonus | 7 year bonus | 10 year bonus | 15 year bonus |
+| --------- | -------- | ------------ | ------------ | ------------ | ------------- | ------------- |
+| junior    | 20,116 € | 21,122 €     | 22,128 €     | 23,134 €     | 24,140 €      | 25,146 €      |
+| `` +20%   | 24,139 € | 25,346 €     | 26,553 €     | 27,760 €     | 28,968 €      | 30,175 €      |
```

Comment:
I suggest to empty all columns except the `annual` one as nobody should stay at the junior position for more than a year

Reply:
Maybe we could also mention that specificity in the title description

Reply:
done!